webapp/player/index.html

```diff
@@ -0,0 +1,215 @@
+<html>
+  <head>
+      <meta name="viewport" content="width=device-width">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/xterm-player@latest/dist/css/xterm-player.min.css" />
+      <script src="https://cdn.jsdelivr.net/npm/xterm@4.4.0/lib/xterm.min.js"></script>
```

Comment:
@nicolesabourin1 I'm not sure how we currently handle the dependencies for the rest of our web frontend, but we'd need to build and include external dependencies alongside the rest of the frontend such that it can work without Internet access. No external CDN delivery can be used for this, as this causes problems for GDPR compliance in addition to breaking things when used without Internet access

Reply:
My understanding is that we should use the `@xterm/xterm` npm package: https://www.npmjs.com/package/@xterm/xterm

Can you confirm? @kristahouse

Reply:
Yes, you can use ES6 module syntax to import modules in a plain JavaScript setup within an HTML page.

Reply:
The problem here is we need a way to host the static web files inside the gateway web server for the xterm js and css files, just like we did for the webapp static files, but in this case the webapp may not be enabled,  @CBenoit will need to make an endpoint for the static files.

Reply:
@pauldumais Just like for the webapp, it’s possible to serve static files via the `GET /jet/jrec/play` endpoint. It serves all the files found in the `player/` folder. Is there anything which should be changed? Do we have different requirements for the player?

Reply:
@CBenoit awesome, I just added the xterm player files locally to the folder